website/static/css/pages/project-page.css

```diff
@@ -142,3 +142,9 @@
 span#nodeDescriptionEditable{
     white-space:pre-line;
 }
+
+pre {
```

Comment:
This selector is too general; it might have side effects for other `pre` tags on the page. Can you apply a CSS class to the specific pre tag you want to style this way?